core/dronecore.h

```diff
@@ -48,26 +47,47 @@ class DroneCore
      * Supports connection: Serial, TCP or UDP.
      * Connection URL format should be:
      * - UDP - udp://[Bind_host][:Bind_port]
-     * - TCP - tcp://[Server_host][:Server_port]
-     * - Serial - serial://[Dev_Node][:Baudrate]
+     * - TCP - tcp://[Remote_host][:Remote_port]
+     * - Serial - serial://Dev_Node[:Baudrate]
      *
-     * Default URL : udp://:14540.
-     * - Default Bind host IP is localhost(127.0.0.1)
-     *
-     * @warning Serial connections are not supported on Windows (they are supported on Linux and macOS).
+     * Default URL : udp://0.0.0.0:14540.
```

Comment:
@julianoes The new interface **must** take a `connection_url` string so presumably this is no longer a default value. Why have we chosen to not have a default anymore?

Reply:
No, the host is not a must, see:
```
- TCP - tcp://[Remote_host][:Remote_port]
- Serial - serial://[Dev_Node][:Baudrate]
- Serial - serial://Dev_Node[:Baudrate]
```

Reply:
Hm, you're right. There is no default. You need to specify it, and that shouild usually be done using a command line argument I suggest.

Reply:
@julianoes Before I do that, this is now inconsistent - the other methods (e.g. for adding UDP) do have defaults. Can you either add the default back OR remote all the other defaults?